Radio.h

```diff
@@ -418,9 +432,21 @@ class CC1101 {
 #else
     spi.strobe(CC1101_SPWD);                // enter power down state
 #endif
+
+    if (PWRPIN < 255) {
```

Comment:
Würde hier lieber 0 für nicht aktiv haben - so wie im AksSinBase::readPin()

Reply:
Ja, das ist doof, weil einige eQ-3 Geräte den Pin 0 verwenden.
Zwar vorwiegend als Taster- und nicht als CC1101-Pin aber 0 ist dennoch ein gültiger/verwendbarer Pin.

Reply:
0 und 1 sind doch RX/TX ? Hm

Reply:
Dann schreib lieber HEX 0xff - allerdings sollten wie das dann auch bei readPin zusätzlich auch 0xff checken. Sonst gibt es unterschiedliche Werte für NICHT BENUTZT.

Reply:
Ich vermute mal wegen der "günstigen" Lage des Pins verwendet eQ-3 die 0 gern.
Bei **allen** Geräten (PSS, WDS, WSC2, BRC8), die ich bisher in den Händen hatte, waren die Pinbelegung identisch.
```
CONFIG_BUTTON 0 (BRC8 = T1)
LED_PIN       8
CC1101_GDO0   2
CC1101_PWR    5
```

Zum Debuggen (serielle Ausgabe) braucht man ja nur den Pin 1; der ist auch meist als Debug-TX Messpunkt auf der PCB ausgeführt.

> dann auch bei readPin zusätzlich auch 0xff checken 

Oder nur auf 0xff ?
_Wobei es recht unwahrscheinlich ist, dass jemand den Pin 0 Enable-Pin benutzen wird._

Reply:
Evtl. doch in ContactState.h sowie PinPosition.h den `en` Pin bei unbenutzt auf `0xff` setzen?
Und dann in der AskSinPP.h bei `readPin` nur auf `0xff` prüfen

Reply:
ich hoffe, wir machen damit nichts kaputt - 0 und 0xff würden erst mal für Kompatibilität sorgen.